.github/workflows/build-and-push-docker-image.yml

```diff
@@ -3,15 +3,18 @@ on:
   push:
     branches:
       - 'dev-cere'
+      - 'feature/import-ddc-pallet'
```

Comment:
Remove before merge to `dev-cere`.

Reply:
Fixed in local branch. I will push the next commit after you merge the PR in ddc-pallet repo ([url](https://github.com/Cerebellum-Network/ddc-pallet/pull/1))